### ISSUE TITLE
Code blocks do not render 88 chars without horizontal scrollbar

### ISSUE THREAD

**Issue by cbrnr** [mne-tools/mne-python#12945]
We're using a line length of 88 characters in our code base, and that's also the standard in the Python community nowadays (e.g., enforced by Ruff and Black). Therefore, I think it would be great if code blocks on our documentation website could be rendered without horizontal scrollbars. For example, see the very last code block at the bottom of https://mne.tools/stable/auto_tutorials/epochs/40_autogenerate_metadata.html.

**Comment by hoechenberger** [comment]
I don't understand the intention here, how would I scroll if you remove the scrollbar? I **have** to scroll on my screen to uncover all of the code

**Comment by hoechenberger** [comment]
Or are you suggesting enforcing shorter lines for code blocks?

**Comment by drammock** [comment]
I thought the intent was "make the content area wider and/or shrink the font so that 88 chars is visible without scrolling". @cbrnr can you confirm/clarify?

**Comment by cbrnr** [comment]
Yes, exactly what you're saying @drammock!

**Comment by hoechenberger** [comment]
yeah but ... on my screen, you cannot make it any wider without hiding the navigation 

slightly reducing the font size should be fine, but I'm not sure of this will free up enough space

**Comment by cbrnr** [comment]
I mean, since we are using 88 characters in our codebase, I think our website should also support displaying 88 characters without scrolling. I'm not a fan of shrinking the font size since this will make the code harder to read, and I would prefer making our main area wider. Forcing a shorter line length for our documentation is also not a good option since we want to stay consistent with our source code.

**Comment by cbrnr** [comment]
Maybe we can make one or both navigation panes a bit narrower so that the main area works with 88 chars in code blocks.
